test-result-summary-client/src/Build/BuildTable.jsx

```diff
@@ -30,13 +30,23 @@ export default class TestTable extends Component {
 
         const renderBuildName = ( value, row, index ) => {
             const resultColor = value.buildResult === "SUCCESS" ? "#2cbe4e" : ( value.buildResult === "FAILURE" ? "#f50" : "#DAA520" );
-            if ( value.type === "Build" ) {
+            if ( value.type === "Build" || value.type === "Perf" ) {
                 if ( value.hasChildren ) {
                     return <Link to={{ pathname: '/buildDetail', search: params( { parentId: value._id } ) }}
-                        style={{ color: resultColor }}> {value.buildName} </Link>;
+                        style={{ color: resultColor }}> {value.buildName} # {value.buildNum} </Link>;
+                } else if (value.tests && value.tests.length > 0) {
+                    let result = value.tests[0].testResult;
+                    if ( value.tests[0]._id ) {
+                        return <div>
+                            <Link to={{ pathname: '/output/test', search: params( { id: value.tests[0]._id } ) }}

```

Comment:
Related issue for this: https://github.com/AdoptOpenJDK/openjdk-test-tools/issues/70

This looks like is a bug with the old design since we would only show the the output of the first iteration (value.tests[0]._id) when the user clicks on a Jenkins build even though that Jenkins build may have multiple iterations, which won't be displayed. This problem will be fixed with the new design.

We can address this issue in #70.

Reply:
Sure.

Reply:
I agree, I have tested it using FVT (non-perf) build type before making this pull request. But since I am having a problem logging to the internal Jenkins server, I would test it again after fixing the logging problem.:)

Reply:
As I noticed, the value type could be "Build" "Perf" and "Test', so if the first condition(either "Build" or "Perf") is not satisfied, it will go to the allTestInfo.